docs/guide/setup/index.rst

```diff
@@ -1,40 +1,42 @@
+.. _userguide_setup:
+
 Simulation Setup
 ================
 
-This section provides details on how to set up free energy calculation or MD simulations.
+This section provides details on how to set up a free energy calculation or MD simulations.
 
 All protocols in OpenFE follow the same general structure: 
 
-* Reading in input structures and Creating ``ChemicalSystem`` \s
-* Defining the Protocol with specific ProtocolSettings
-* Creating ``LigandAtomMapping`` \s for relative free energy calculations Protocols
+* Reading in input structures and creating :class:`.ChemicalSystem` \s
+* Defining the :class:`.Protocol` with specific `ProtocolSettings`.
+* Creating :class:`.LigandAtomMapping` \s for relative free energy calculation `Protocols`.
 
 .. image:: img/setup_1x.png
-   :width: 60%
+   :width: 70%
    :align: center
    :alt: Concept of a ChemicalSystems and Transformations
 
-The image below demonstrates how, for relative free energy calculations, you plan a network of ligand transformations starting from input SDF / MOL2 / PDB files:
+The image below demonstrates how, for relative free energy calculations, you plan a
+network of ligand transformations starting from input SDF / MOL2 / PDB files:
 
 .. image:: img/setup_2x.png
-   :width: 60%
+   :width: 70%
    :align: center
    :alt: Concept of a LigandNetwork and AlchemicalNetwork
 
 The procedure for setting up a simulation depends somewhat on the on the
```

Comment:
```suggestion
The procedure for setting up a simulation depends somewhat on the
```
Also "somewhat" here sounds kind of vague, do we want to be a little more specific or just say it depends on it (no somewhat)?